source

```diff
@@ -99928,16 +99958,29 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
 
     <tr>
      <td><dfn data-x="parse-error-disallowed-content-in-noscript">disallowed-content-in-noscript</dfn>
-     <td><p>This error occurs if the parser encounters content in a <code>noscript</code> element
-     that is not allowed by it's content model. In such a case, the parser implicitly closes the
-     <code>noscript</code> element and processes content as if it was placed after the
-     <code>noscript</code> element in the markup.
+     <td><p>This error occurs if the parser encounters any content other than <span>ASCII
+     whitespace</span>, <span data-x="syntax-comments">comments</span>, a <code>basefont</code>
+     element, a <code>bgsound</code> element, a <code>link</code> element, a <code>meta</code>
+     element, a <code>noframes</code> element, or a <code>style</code> element within a
+     <code>noscript</code> element. In this case, the parser implicitly closes the
+     <code>noscript</code> element and processes the content as if it was placed after the
+     <code>noscript</code> element in the markup.</p>
 
     <tr>
      <td><dfn data-x="parse-error-duplicate-attribute">duplicate-attribute</dfn>
      <td><p>This error occurs if the parser encounters an <span
      data-x="syntax-attributes">attribute</span> in a tag that already has an attribute with the
-     same name. The parser ignores all such duplicate occurrences of the attribute.
+     same name. The parser ignores all such duplicate occurrences of the attribute.</p>
+
+    <tr>
+     <td><dfn data-x="parse-error-misplaced-start-tag-for-head-element">misplaced-start-tag-for-head-element</dfn>
+     <td><p>This error occurs if the parser encounters a <span data-x="syntax-start-tag">start
+     tag</span> whose tag name is "head" that is not the first start tag within the
```

Comment:
Exception: `<html><html><head>` does not trigger this error. Maybe again say that `<head>` is encountered where it is not expected?

Reply:
> where it is not expected

I find it a bit vague, since descriptions IMO should suggest how error can be fixed and such wording doesn't really clarify which position for `head`is allowed.